README.md

```diff
@@ -1,2 +1,43 @@
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+| **Platform Launch**  | Platform  | The reference to Launch is not always necessary if the sentence was referring to the service provided. In this case Launch can be omitted as tags is a set of capabilities and not a service. Here, Platform provides the functionality.  | **Platform Launch** passes `settings` into the rule’s condition-type library modules where conditions are then evaluated. | **Platform** passes `settings` into the rule’s condition-type library modules where conditions are then evaluated. |
+| **Platform Launch** | {ommitted} | If context has already been established, naming the capability in the sentence may be obsolete and can be removed entirely. | If the extensions, tags, and systems you are going to deploy through **Platform Launch** are very similar across your sites or applications, you might want to include them in the same property. | If the extensions, tags, and systems you are going to deploy are very similar across your sites or applications, you might want to include them in the same property. |
+| **An Adobe Experience Platform Launch {noun}** | a tags-enabled {noun} | When tags is providing the functionality to the noun, the hyphenated 'enabled' adds clarity to the sentence.  | This extension provides the functionality for adding the 'Media' tracker instance to **an Adobe Experience Platform Launch site** or project.  | This extension provides the functionality for adding the `Media` tracker instance to **a tags-enabled site** or project.   |
+| **The Launch runtime**  |  The tag runtime | Use singular version of Tag and "The" as an article when talking about a general concept/process rather than individual runtime events.  | **The Adobe Experience Platform Launch runtime** (delivered to clients' websites) is designed to support the following browsers:  | ****The tag runtime**** in Adobe Experience Platform (delivered to clients' websites) is compatible with multiple browsers:  |
+| **A Launch property**  | A tag property  | Use the singular version of 'tag' when writing "tag {noun}" in a sentence. | When users install an extension to an Adobe Experience Platform **Launch property**, they will be shown the extension configuration view which your extension will provide.  | When users install an extension to **a tag property**, they will be shown the extension configuration view which your extension will provide.  |
+| **the {noun} extension in Platform Launch**  |  the {noun} tag extension in Platform | The name of the extension will precede the tag qualifier. Later references to tags (Launch) are removed once context has been established.  | Learn about **the Adobe ContextHub extension in Adobe Experience Platform Launch**. | Learn about **the Adobe ContextHub tag extension in Adobe Experience Platform**.  |
```

Comment:
Let's organize these better...these are just more examples of how to handle "Launch {noun}". Gives these their own section.